lib/cdo/db.rb

```diff
@@ -1,6 +1,7 @@
 require 'sequel'
+require 'sequel/connection_pool/threaded'
 
-def sequel_connect(writer, reader)
+def sequel_connect(writer, reader, validation_frequency = nil)
```

Comment:
Please add a function header explaining the params (`@param`), and add a link to the [sequel validation_frequency explanation](http://sequel.jeremyevans.net/rdoc-plugins/files/lib/sequel/extensions/connection_validator_rb.html).

This might be more readable as a keyword argument: `validation_frequency: nil`. Then callers would specify it by name, e.g.
```ruby
sequel_connect(CDO.devinternal_db_writer, CDO.devinternal_db_writer, validation_frequency: -1) 
```